### DISCUSSION TITLE
Has there been any progress in setting up a volunteer Telegram support group?

### DISCUSSION THREAD

**Discussion by David263** [yagop/node-telegram-bot-api#1046] [General]
I have set up a new Telegram account, and I'm having a problem with it. I can't find an active support group for Telegram anywhere. This would be a good thread for someone to announce such a group.

**Comment by danielperez9430** [comment]
Hi @David263 , we have a group in Telegram

🔗 NTBA Support: https://t.me/+nc3A9Hs1S81mYzdk

**Reply by dhruv160410116084** [reply]
Hi @danielperez9430 this link doesn't work

**Reply by danielperez9430** [reply]
> Hi @danielperez9430 this link doesn't work

Test with this other link: https://t.me/+nc3A9Hs1S81mYzdk

I also reactivate the other, so maybe now both works now
